### PR TITLE
Xenobiology consoles can no longer manipulate slimes.

### PR DESCRIPTION
**Pull request by Pickle-Coding**
## About The Pull Request
Removes all slime and monkey manipulation behaviour from the xenobiology consoles.
## Why It's Good For The Game
So slimes have an actual presence when xenobiologists interact with them. Should be more fun, engaging and interesting this way.
## Changelog
:cl:
balance: The xenobiology consoles can no longer manipulate slimes and monkeys.
/:cl:

### CODE REVIEW

**Comment by moocowswag** [comment]
This is a waste of gbp since someone is already reworking xenobio.

**Comment by dwasint** [comment]
this requires remapping to add chutes for monkeys as there is no more monkey manipulation also someone is reworking xenobiology on tg.

**Comment by Namwty** [comment]
POV instead of spending 40 minutes on terry trying to get a crossbreed. ( Escape shuttle is already docked and ready to depart )

We will be spending 120 minutes trying to crossbreed slimes, cause RNG was shit and I had to spend 10+ minutes looking at a disposals loading bar.


But jokes aside, **it was a thing before for long time**, I doubt that anyone who actively plays/ed xenobiology would find the "old times" fun, especially cause crossbreeds and what not exists. Like this whole "interacting with slimes" will be fun for first 5+ times, after it's literally "I'll never touch xenobiology again".


Biggest issue is, RNG, it's already ass, so either lessen RNG aspect of xenobiology or don't have this. Last thing TG needs is people sitting in one room for 3 hours with 0 interactions, cause they spend 50% of time either dealing with RNG or "interacting with slimes" ( that aren't even players ).

**Comment by Helg2** [comment]
Take your meds, please.

**Comment by Elli-Skala** [comment]
Replace the glass pens with force fields and add a slime gun then we could be actually talking

**Comment by Livrah** [comment]
I think we should replace xenobio with fishing

**Comment by GuillaumePrata** [comment]
Xenobio has been "being reworked guys don't worry" since I joined the game almost 5 years ago.

At this point I think we should just start shooting up and see what lands where, merge this PR, see which issues pop up, fix them, lower RNG because it is a dumb timegate, see what issues pop up, repeat.

Xenobio is boring in lab and has nothing interesting outside the lab until the time gate is achieved, so we either make it less boring, by adding small things like this, we cut the time gate, or we wait another 5 years for Xeno Jesus to save us.

**Comment by nianjiilical** [comment]
im not going to argue that xeno isnt one dimensional and needs work but "should be more fun and engaging" sounds extremely wrong when this is how xeno used to work and all it did was make the same job take way way longer

its the same logic as saying botany is one dimensional so we should get rid of hydro trays and make botanists only use dirt piles to keep them on their toes

**Comment by Draggeru** [comment]
This is dumb because you're not changing the intended interaction with xenobiology. The console interaction is because the slime and monkey management would be a nightmare.

Don't remove an interaction without proposing an alternative, that's game design 101

**Comment by GuillaumePrata** [comment]
> was make the same job take way way longer

And that is my whole other point, we can speed up Xenobio multiple ways, we just choose to make it in the most boring way possible years ago and never reverted a bad decision.

**Comment by Sakamoto4ka** [comment]
Slime rancher

**Comment by Iamgoofball** [comment]
hi, as someone who actually played xenobio back in the day, lmao no fuck ever going back to manually shoving slimes in/out of pens, jesus

**Comment by the-orange-cow** [comment]
> > was make the same job take way way longer
> 
> And that is my whole other point, we can speed up Xenobio multiple ways, we have just chosen the most boring way possible years ago and never reverted a bad decision.

Then an appropriate fix should be implemented before taking away such an important quality of life feature.  Removing slime console functionality doesn't make it fun, it makes it exceedingly unfun.

**Comment by necromanceranne** [comment]
I agree with goof, tedium doesn't solve xenobio's issues.

**Comment by atlasle** [comment]
> This is a waste of gbp since someone is already reworking xenobio.

really? who?

**Comment by SimplySmiley123** [comment]
> > was make the same job take way way longer
> 
> And that is my whole other point, we can speed up Xenobio multiple ways, we have just chosen the most boring way possible years ago and never reverted a bad decision.

Probably because trying to interact with slimes and especially monkeys now _is utter ass._
On top of how long everything takes anyway despite the consoles existing.

**Comment by GuillaumePrata** [comment]
> Then an appropriate fix should be implemented before taking away such an important quality of life feature

Yes... Like I said... That is my whole other point...

**Comment by GuillaumePrata** [comment]
> On top of how long everything takes anyway despite the consoles existing.

Again... my whole other point....

C'mon guys we can all read here right?

Do this, triple or quintuple the amount of slime cores each slime produces, we have so many balance levers, no one should be complaning about how slow xenobio as a counter point when it is so easy to speed up the time gate with very simple PRs.

**Comment by RedBaronFlyer** [comment]
"Should be more **fun**, **engaging** and **interesting** this way."
Okay, but like, how specifically?

**Comment by Pickle-Coding** [comment]
The idea is xenobiologists might accidentally slap a slime, making them angry and causing a slime invasion all over the station.

I could compensate by making slimes breed faster or something. Really I just don't see how a console that does all the work without risk is engaging to be honest.

**Comment by moocowswag** [comment]
This doesnt make xenobio less boring it makes it tedious.

You can't really make xenobio less boring / give it more to do because the job that gets xenobio also gets circuits and ordnance and cytology.

How do these sub departments fit into a single job (scientist)? Because you are meant to juggle them, you can feed the slimes then leave to ordnance and refine anomaly cores, then upgrade the parts to xenobio and feed the slimes again, then go do circuits or experisci shit to speed up research, then go to xenobio and feed the slimes again and maybe grab an extract or two, then look into making something with your refined anomaly cores, then go do xenobio again.


Xenobio is not a standalone commitment, nothing that scientists do are standalone commitments because they were not designed with that in mind, each and every single thing a scientist has access to requires bursts of effort with a ton of downtime.

**Comment by GuillaumePrata** [comment]
Moocow is right on this one.

Making slimes breed faster would make scientist need to babysit them more, making them drop more cores, or multiply into more slimes at once would increase the speed of the department without decreasing the downtime.

**Comment by moocowswag** [comment]
And if we were to decrease the downtime of xenobio we would have to look towards decreasing the downtime of the rest of the sci department.

If xenobio has no downtime you end up with 1 or 2 scientists hogging it all round and then the rest of the 3-4 scientists are just afking at a research console after doing ordnance and the experiments that are accessible to them.

We would have to either go back to active point generation or more than double the amount of experiments and discounts available to provide an actual gameplay loop for scientists rather than soft locking everything behind waiting at a console for 20 fucking minutes.

**Comment by Pickle-Coding** [comment]
Guess I'll make slimes multiply into 4 or something instead of speed up breeding. I don't really know anything about the slime core anomaly, but I thought that most of the time spent is trying to get the right mutation.

**Comment by Profakos** [comment]
I should mention that I am turning slimes into basic mobs right now, i need at least a week or two make it ready for PRing, due to the complexity of the subject and also me being busy for a few days, the coder actually working on the rework is waiting on me, sorry :pensive: I will keep an eye on this so i can carry over the change if this gets merged.

**Comment by ZeroKelvn** [comment]
Ok I may as well throw my hat in the ring as a cringe Xenobiology main on Manuel.

For context, the usual round of Xenobiology starts with feeding the slimes, running ExperiSci for the stock part experiments, to get Tier 3, then babysitting the slimes for the rest of the shift. Managing the monkey count and making extra slime cores takes up enough time to fill in between generations as one person in the early game, and the mid game is taken up with preparing industrial-grade farming. Late game usually sees Rainbow slimes at the hour mark if nothing goes wrong and the shift makes it far enough, and going from there. I don't really consider any of it 'dull', since there's always something you CAN be doing if you crossbreed well.

From my perspective, the RNG of slime splitting is the biggest time sink that Xenobiology takes. This can be controlled with slime mutation potions, though the one that DECREASES mutations is on a Tier 1 slime, and the one that INCREASES mutations is on a Tier 4, which could take over half an hour to get if you're unlucky while playing eugenics with the slimes. And when you're working with a player base that seems to consider a 90 minute shift long, you're already on borrowed time. Every moment winds up precious.

I don't understand what this PR is actually trying to accomplish. Slimes consider the person that feeds them their friend (or are supposed to), so they're not going to attack if they have to go into the pen. The only issue comes from if they're neglected and hungry, at which point they'll already start smashing out of the pens and cause problems. All this does is create extra busywork in a subdepartment that already doesn't have enough time to function fully, and make a Revenant more likely because nobody is gonna be assed to traipse in there to recover the corpses, on top of needing to redesign North Star and Birdshot (the worst maps for Xenobio already) due to modern design not needing the chutes. 

If you want to make Xenobio more engaging, my suggestion would be to give it more interplay with other departments. A good example I can think of is the Stabilized Light Pink, which is a good boon for Paramedics, or (failing in execution) the Green extract using Radium for Lizard Mutation Toxin needing Chem's help. I've been eyeing the Crystalizing (Adamantine) Crossbreeds for a long while as a way to help the station a lot, but I don't know any BYOND code currently and would probably have to add the ones leading up to it first. But just giving Xenobio something to aim for that doesn't amount to one-off murder implements would probably help.

tl;dr, this is simply anti-QoL and we should try to add instead of subtract.

**Comment by Profakos** [comment]
Subtracting will happen anyways, as crossbreeds will be replaced with different mechanics in the wip rework.

**Comment by carlarctg** [comment]
This PR is so incredibly deaf to what xenobiology is and how people actually interact with it it's honestly amazing. The computer interactions are simply a necessity because it would be fucking hellish otherwise. Imagine chemistry but you have to manually put the chemicals in each beaker, dripping them from a large jug. Why are they all your PRs like this?

**Comment by GuillaumePrata** [comment]
> Imagine chemistry but you have to manually put the chemicals in each beaker, dripping them from a large jug.

It's funny to bring this up because people often comment about how more engaging it is to do ghetto chemistry than it is to do dispenser/factory.

The computer interactions are a necessity because we simply never bothered to add other more meaningful QoLs to the department, it's the same issue I have with the RCD.

Eons ago we all agreed that reconstruction was too much, but instead of slowly polishing the system, we created a magic item that makes everything easy, even if it is completely boring and block future interesting interactions.

The xeno console is basically the same, it has no reason to exist because we can simply polish Xenobio to be a better department without it.

**Comment by san7890** [comment]
this doesn't solve any of the problems with xenobiology (which is that it's a very surface deep job) while increasing the boring tedium parts of it. it just makes a bad thing _worse_. that's not a good solution.

if you're curious as to what we would like to see be added to xenobiology at the bare minimum, you may look at what our head coders have written here: https://hackmd.io/@tgstation/HyOAnhDPI
